requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+arrow
+boto
+boto3
+celery==4.0.2
+django>=1.11,<2.0
+jsonfield==1.0.3
+mbq.metrics==0.4.1
+rollbar
```

Comment:
Should most of these be in requirements.txt, not requirements-dev.txt?

Reply:
yea i'm going to revisit dependencies, i'm not sure which one should be where right now. will give it a thought after i get tests to works